iOS/docs/kickoff/configure-ci/README.md

```diff
@@ -0,0 +1,171 @@
+```
+
+### Decryption
+
+Add this as first step of the `bitrise.yml`:
+
+```
+    - script@1.1.3:
+        title: Decrypting files...
+        inputs:
+        - content: openssl aes-256-cbc -k "$ENCRYPT_PASSWORD" -in secrets.tar.enc -out secrets.tar -d && tar xvf secrets.tar
+```
+
+Don’t forget to include `ENCRYPT_PASSWORD` as explained in [Configure App Environment variables](#configure-app-environment-variables)
+
+## Create Bitrise app...
+
+### Log in to [Bitrise](https://www.bitrise.io/) using your Wolox's github account.
+
+If you’re not added to the `Wolox` organization, you should ask the technical leader to do so.
+
+### Add application
+
+In Bitrise dashboard, press "Add new app" to start the process. 
+
+### Choose project’s repository
+
+Before starting, make sure `Wolox` is selected in the "Add new app to" option in top left.
+
+![choose organization](./resources/br1.png)
+
+From the list of platforms, choose `Github`. Below, in the left part of the screen you can see a list of github organizations you contribute to. Choose `Wolox` (or the organization the repository is under), and then at the right choose the repository you want to integrate.
+
+![choose project](./resources/br2.png)
+
+### Setup repository access
+
+Select `auto-add SSH Key`. The key should be added automatically after a moment. If it doesn’t work, you may need to do it manually.
+
+NOTE: Only Github admins can automatically add the SSH key in this step. In case you have no access, skip this step and ask your TL to do it.
+
+![choose project](./resources/br3.png)
+
+### Validation setup
+
+Choose the main branch of your project (master or development usually). Bitrise will scan the branch to check for the correct configurations. This takes about 5 minutes.
+
+### Project build configuration
+
+Select iOS. Verify that the correct XCode stack is selected and confirm.
+
+![choose project](./resources/br4.png)
+
+### Webhook setup
+
+Select Register a new webhook for me. It will allow Bitrise to know when a `push` / `pull request` has been made, so it can test it. This step can only be done by an admin.
+
+NOTE: Only Github admins can automatically add the webhook in this step. In case you have no access, skip this step and ask your TL to do it. It can be done from the Code tab in the project’s dashboard. Under “webhooks”, there’s a webhook URL which you can add to your Github repository following [this guide](https://github.com/bitrise-io/bitrise-webhooks#github---setup--usage).
+
+### Finish configuration
+
+Select We’ve kicked off your first test build for you to continue. Bitrise will run a build at this step, you should cancel it since it runs with another configuration, thus it will fail.
+
+![choose project](./resources/br5.png)
+
+## Configure Bitrise’s `bitrise.yml`
+
+When Bitrise builds trigger, they look for the configuration in the web’s `bitrise.yml`. That file will call the one from our repository. 
```

Comment:
This* file should* call the one from our repository. 

Reasoning:
- The file was just mentioned.
- It doesn't, it should be configured to do so. (And then 'For this' makes sense on the next paragraph.)